ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,57 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class OCSInitialization(NamespacedResource):
+    """
+    OCSInitialization object.
+
+    API reference (source code linked here due to lack of API doc for this resource):
+        https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/ocsinitialization_types.go
+
+    Args:
+        enable_ceph_tools : Boolean
+            When set to True, creates a Deployment named "rook-ceph-tools" in the "openshift-storage" namespace.
+            More information about the Rook Ceph Toolbox can be found here:
+            https://rook.io/docs/rook/v1.10/Troubleshooting/ceph-toolbox/
+    """
+
+    api_group = NamespacedResource.ApiGroup.OCS_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=False,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        enable_ceph_tools=None,
+        **kwargs,
+    ):
+        if not enable_ceph_tools:
```

Comment:
Please move it inside `to_dict()` after `if not self.yaml_file:`

Reply:
Done